### PR TITLE
Reboot command for fake VMs

### PR DESCRIPTION
**Pull request by rtimush**
Accepts delay as a parameter, returns some message.

### CODE REVIEW

File: fake-vm-component-scripts/fake-vm/reboot.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+import sys
+import yaml
+import fvm
```
Comment (netvl) [inline fake-vm-component-scripts/fake-vm/reboot.py]: Not used?

Comment (rtimush) [inline fake-vm-component-scripts/fake-vm/reboot.py]: Fixed

**Comment by rtimush** [comment]
@netvl up

**Comment by netvl** [comment]
Okay, looks good.

**Pull request merged**